text_collector_examples/deb_check_package_updates.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+/usr/bin/apt-get update
```

Comment:
Yep, this is tricky; but at systems without unattended-upgrades(pure debian, or ubuntu with disabled auto) you would see none changes in available to update packages, since no trigger apt-get update.